.github/workflows/clair-in-ci.db.yaml

```diff
@@ -0,0 +1,79 @@
+name: Clair in CI DB - test
+
+# This workflow tests latest database version of clair-in-ci and
+# pushes it to app-studio registry in case all tests passed
+
+on:
+  repository_dispatch:
+    types: trigger_workflow
+    paths:
+      - clair-in-ci/**
+      - .github/workflows/clair-in-ci-db.yaml
+
+env:
+  REGISTRY: quay.io/redhat-appstudio
+  IMAGE_NAME: clair-test-build
+  LATEST_TAG: latest
+  PREVIOUS_TAG: previous
+  TO_TEST: ${{ github.event.client_payload.tag }}
+
+jobs:
+  test-and-push:
+    name: Test and push image
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Dockerfile linter
+        if: ${{ github.event.client_payload.event == 'pull_request' }}  # don't break regular rebuilds if linter is updated
+        uses: hadolint/hadolint-action@v3.1.0
+        with:
+          dockerfile: clair-in-ci/Dockerfile
+          ignore: DL3041
```

Comment:
IMO it makes sense, to have these steps in separate workflow (we can call it linters.yaml) and not combine it with .tekton

Reply:
no, it belongs here and takes 10 minutes

Reply:
Separated and added.